test/shared/src/main/scala/zio/test/ZSpecRunner.scala

```diff
@@ -19,7 +19,7 @@ package zio.test
 import zio._
 
 trait ZSpecRunner {
-  def run[E, L](spec: ZSpec[Any, E, L]): UIO[ExecutedSpec[Any, E, L]]
+  def run[R, E, L](spec: ZSpec[R, E, L]): RIO[R, ExecutedSpec[R, E, L]]
```

Comment:
And looks like I made yet another error: instead of `RIO` we can use `ZIO[R, Nothing, ...]` because running cannot (should not) fail.

Sorry about that!